packages/sensors/example/lib/snake.dart

```diff
@@ -73,12 +73,14 @@ class SnakeState extends State<Snake> {
   void initState() {
     super.initState();
     accelerometerEvents.listen((AccelerometerEvent event) {
```

Comment:
The return value of listen is a `StreamSubscription<AccelerometerEvent>`. That should move into a `_accelerometerEventSubscription` field. Please `cancel()` it in the `dispose` method.